package.json

```diff
@@ -46,6 +46,7 @@
     "@kyper/utilityrow": "^2.1.0",
     "@mui/icons-material": "^6.1.5",
     "@mui/material": "^6.1.5",
+    "@mxenabled/mxui": "^1.2.0",
```

Comment:
I have [this PR](https://github.com/mxenabled/connect-widget/pull/182) that switches from `@kyper/mui` to `@mxenabled/mxui`. Since `@mxenabled/mxui` is the maintained package going forward, maybe we can coordinate and merge our efforts.